sources/us/la/red_river_parish.json

```diff
@@ -0,0 +1,22 @@
+{
+    "coverage": {
+        "US Census": {"geoid": "22081", "name": "Red River Parish", "state": "Louisiana"},
+        "country": "us",
+        "state": "la",
+        "county": "Red River"
+    },
+    "data": "http://www.redriver911.com/wp-content/uploads/2016/03/911_ADDRESS_Public.zip",
+    "website": "http://www.redriver911.com/data/",
+    "type": "http",
+    "accuracy":"1",
+    "compression": "zip",
+    "conform": {
+        "type": "shapefile",
+        "number": "NUMERICAL",
+        "street": "Street",
+        "city": "Add_City",
+        "postcode": "Add_Zip",
+        "lon":"Logitude",
```

Comment:
We automatically extract the latitude and longitude from shapefiles, so the `lat` and `lon` keys should be removed in this case.